dbt_subprojects/daily_spellbook/models/_sector/airdrops/zksync/projects/zksync_zksync_airdrop_claims.sql

```diff
@@ -0,0 +1,53 @@
+{{
+    config(
+
+        schema = 'zksync_zksync',
+        alias = 'airdrop_claims',
+        materialized = 'incremental',
+        file_format = 'delta',
+        incremental_strategy = 'merge',
+        unique_key = ['recipient', 'tx_hash', 'evt_index'],
+        post_hook='{{ expose_spells(\'["zksync"]\',
+                                "project",
+                                "zksync",
+                                \'["lgingerich"]\') }}'
```

Comment:
```suggestion
        unique_key = ['recipient', 'tx_hash', 'evt_index']
```

Reply:
can we remove this post hook? we should only expose the final `airdrop.claims` in data explorer to keep clean

Reply:
this is a common theme we're trying to work towards